### DISCUSSION TITLE
flux1-schnell-bnb-nf4 support?

### DISCUSSION THREAD

**Discussion by Yogesh-DevHub** [runew0lf/RuinedFooocus#163] [General]
the wiki says it supports black-forest-labs/FLUX.1-schnell can it be for flux1-schnell-bnb-nf4?

**Comment by yownas** [comment]
Not yet. That is just the more experimental part using diffusers. But these two models should work (just place the safetensors files in models/checkpopints):

https://civitai.com/models/623224/flux1s-16gb
https://civitai.com/models/623525/flux1d-16gb

I'm trying to figure out how ComfyUI handle gguf-files and hopefully will add this soon™.

**Reply by Yogesh-DevHub** [reply]
I tried both the models, but it is generating blur images. It is like, it just stops after first or second step leaving the blur/noisy image as it is, can you have a look here. 
https://youtu.be/TfC8N4LzFSY

Thank you.

**Comment by yownas** [comment]
It might be that you need to make a new Performance setting. Flux can be a bit picky with CFG and Sampler for example.

This is what I use for Flux.1 Dev:
![Screenshot from 2024-08-23 18-24-53](https://github.com/user-attachments/assets/ed026d3b-5045-4849-a24a-ec24c6593f90)

...and this for Flux.1 Schnell
![Screenshot from 2024-08-23 18-23-59](https://github.com/user-attachments/assets/3d1756ef-2f73-423b-883f-e265a773731c)

(These are probably not "the best" settings, but they should be a good start.)

**Reply by Yogesh-DevHub** [reply]
It works, thank you. I will be using for few days now.

I am trying to get Fooocus like applications which supports flux, I made a video on "metercai/SimpleSDXL", but it freezes sometimes so I was searching for other forks and found this. 

RF has some interesting features (PowerUp), over what Fooocus offers. 
I don't think PowerUp works with flux, am I correct?
Will RF be able to use LORA for flux? Are there any plans?

**Reply by yownas** [reply]
Sorry, no, most of the PowerUps will only work with SDXL models. But it might be something we take a look at later to see if we can find good controlnets for Flux.

Loras for Flux should work. I haven't tried them a lot, but you can find some on [civitai.com](https://civitai.com/) (make sure to filter for Loras for the correct Flux-version.)
